src/Adapter/Product/Repository/ProductPreviewRepository.php

```diff
@@ -1,99 +0,0 @@
-<?php
-/**
- * Copyright since 2007 PrestaShop SA and Contributors
```

Comment:
A removed file?

Reply:
Yes the method was moved in another repository